packages/@sanity/portable-text-editor/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "rootDir": "./src",
     "outDir": "./dist/dts",
     "jsx": "react",
-    "types": ["jest", "puppeteer"]
+    "types": ["jest", "puppeteer", "node"]
```

Comment:
Is the `types` array actually needed here? I believe TypeScript will pick up these automatically, if they are installed as `@types/*`.

Reply:
They are for running the local dev server (only).

Reply:
Haven't tried removing it all together though!

Reply:
Seems to be working fine when I remove the entry altogether.